### PR TITLE
Initial tests for Dark launch haxm upgrade

### PR DESCRIPTION
**Pull request by daneb**

### CODE REVIEW

File: test/ruboto_setup_test.rb [hunk 1]
```diff
@@ -1,22 +1,43 @@
 require File.expand_path('test_helper', File.dirname(__FILE__))
+require 'ruboto/util/setup'
```
Comment (donv) [inline test/ruboto_setup_test.rb]: Is this require needed?  Is it not required in test_helper.rb?  No biggie.

**Comment by donv** [comment]
@daneb looks good, but travis is failing.

**Comment by donv** [comment]
@daneb I guess this is the one:  https://github.com/ruboto/ruboto/blob/master/lib/ruboto/util/setup.rb#L613

The case for LINUX is missing.  You got it?

**Comment by daneb** [comment]
Thanks ;)

**Comment by donv** [comment]
I just pushed a small commit.  Just cancel the travis build for my build if you need the build time for this PR.

**Comment by daneb** [comment]
Hi @uwe

I just pushed again into that same branch with the Intel HAXM upgrade
change.
Tested and looks good.

Let me know your thoughts and I added the LINUX case.

Dane

**Comment by donv** [comment]
:+1:

**Pull request merged**